sockeye/model.py

```diff
@@ -198,6 +203,23 @@ def encode_and_initialize(self, inputs, valid_length=None, constant_length_ratio
 
         return states, predicted_output_length
 
+    def embed_and_encode(self, source, source_length, target, target_length):
```

Comment:
To make pylint happy, can we rename this to `_embed_encode()` and do the following in the constructor:
```
self.embed_and_encode  = self._embed_and_encode()
if self.forward_pass_cache_size > 0:
    self.embed_and_encode = self._cache_wrapper(self._embed_and_encode)
```

Reply:
Thanks for the suggestion! I've updated accordingly.